src/components/card/card.module.scss

```diff
@@ -7,23 +7,34 @@
   flex-wrap: wrap;
 }
 
-.styledImage {
-  border-radius: 3%;
-  margin-bottom: 1.5rem;
+.imageContainer {
+  position: relative;
+  padding-top: 56.6%;
+  display: block;
   width: 100%;
   height: 15rem;
+  margin-bottom: 1.5rem;
+}
+
+.styledImage {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  border-radius: 3%;
+  object-fit: fill;
 
   @media screen and (max-width:50rem) {
-    height: 8.125rem;
+    height: 14rem;
   }
```

Comment:
Does it work in mobile?

Reply:
Yes

Reply:
I was referring to this prop breaking the 16:9 ratio, is that ok?

Reply:
If I delete the media, nothing happens and the value 16:9 ratio is okay on mobile.